Bottom: b29e42e4d06201c76845a1ea903181a8e37d64f7
Top:    028ec64ac8a1af4bb2e94fb65df25bf64d320263
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 14:34:51 +0100

Refresh of move-c-gratuitous-cosmetic-cha

---

diff --git a/src/move.c b/src/move.c
index 7ba7476..ecc5928 100644
--- a/src/move.c
+++ b/src/move.c
@@ -569,7 +569,7 @@ moveline_callback (const BoxType * b, void *cl)
 }
 
 static void *
-MoveLineToLayer (LayerTypePtr Layer, LineTypePtr Line)
+MoveLineToLayer (LayerType *Layer, LineType *Line)
 {
   struct via_info info;
   BoxType sb;
@@ -641,8 +641,8 @@ MoveLineToLayer (LayerTypePtr Layer, LineTypePtr Line)
  * moves a text object between layers; lowlevel routines
  */
 static void *
-MoveTextToLayerLowLevel (LayerTypePtr Source, TextTypePtr Text,
-			 LayerTypePtr Destination)
+MoveTextToLayerLowLevel (LayerType *Source, TextType *Text,
+			 LayerType *Destination)
 {
   TextTypePtr newone = GetTextMemory (Destination);
 
@@ -672,7 +672,7 @@ MoveTextToLayerLowLevel (LayerTypePtr Source, TextTypePtr Text,
  * moves a text object between layers
  */
 static void *
-MoveTextToLayer (LayerTypePtr Layer, TextTypePtr Text)
+MoveTextToLayer (LayerType *Layer, TextType *Text)
 {
   TextTypePtr newone;
 
@@ -700,8 +700,8 @@ MoveTextToLayer (LayerTypePtr Layer, TextTypePtr Text)
  * moves a polygon between layers; lowlevel routines
  */
 static void *
-MovePolygonToLayerLowLevel (LayerTypePtr Source, PolygonTypePtr Polygon,
-			    LayerTypePtr Destination)
+MovePolygonToLayerLowLevel (LayerType *Source, PolygonType *Polygon,
+			    LayerType *Destination)
 {
   PolygonTypePtr newone = GetPolygonMemory (Destination);
 
@@ -748,7 +748,7 @@ mptl_pin_callback (const BoxType *b, void *cl)
  * moves a polygon between layers
  */
 static void *
-MovePolygonToLayer (LayerTypePtr Layer, PolygonTypePtr Polygon)
+MovePolygonToLayer (LayerType *Layer, PolygonType *Polygon)
 {
   PolygonTypePtr newone;
   struct mptlc d;
